cdc_rsync/zstd_stream.h

```diff
@@ -36,13 +36,13 @@ class ZstdStream {
   // Sends the given |data| to the compressor.
   absl::Status Write(const void* data, size_t size) ABSL_LOCKS_EXCLUDED(mutex_);
 
-  // Flushes all remaining data and sends the compressed data to the socket.
-  absl::Status Flush() ABSL_LOCKS_EXCLUDED(mutex_);
+  // Finishes the stream and flushes all remaining data.
```

Comment:
Can I still send more data once I called `Finish`? The name suggests I might not.

Reply:
I don't think you can, but I don't really remember. The code never does it, and unzstd_stream sets the eof marker if that happens, but I think technically that's just the end of a frame and zstd can send more frames.